helpers/Apip.php

```diff
@@ -62,9 +84,13 @@ static public function extractApipAccessibility(DOMDocument $doc)
      * @param DOMDocument $apipContent
      */
     static public function mergeApipAccessibility(DOMDocument $qtiItem, DOMDocument $apipContent)
-    {
-        $newNode = $qtiItem->importNode($apipContent->documentElement, true);
-        $qtiItem->documentElement->appendChild($newNode);
+    {        
+        $apipXMLString = $apipContent->saveXML($apipContent->documentElement);
+        
+        $fragment = $qtiItem->createDocumentFragment();
+        $fragment->appendXml($apipXMLString);
+        
+        $qtiItem->documentElement->appendChild($fragment);
```

Comment:
what is the difference between this version (saveXML+createDocumentFragment+appendXml) and the previous one (importNode+appendChild)